### DISCUSSION TITLE
Saturday 26th June 2021

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#1906] [Community Call]
Welcome to our multi community collab call 🎉 

We have 2 threads
- Introductions
- Your projects

Any questions feel free to start a new thread below, also reply to other people's questions 🤓 

**Collaboration 1st, code 2nd**

**Comment by eddiejaoude** [comment]
Leave your introductions here... 👇

**Reply by KhushiSharma22** [reply]
Hi there :) I am Khushi Sharma, almost a Computer science Junior and I am from India. I love playing around Android and Kotlin on the technical front, and I love contributing to the community in every way possible.

**Reply by GowthamPB** [reply]
Hi everyone my name is Gowtham. I am a second year engineering student from India. I am interested in web development and graphic design. I recently joined EddieHub :)

**Reply by ItsMeIshan** [reply]
Hey everyone! Glad to introduce myself to all folks 😄 i am a 3 year student of B. tech learning web development with JS!

**Reply by ish-u** [reply]
Hey everyone my name is Anmol. I am 3rd year BSc Student from India. I like learning new things.  I am currently experimenting with Spotify Web API.

**Reply by Lilykhan786** [reply]
Hello! I'm Lily Khan, a front-end developer from India. I'm currently in high school and I like Css, designing and bullet journaling. :D I'm really happy to be a part of EddieHub community! <3

**Reply by Ciggzy1312** [reply]
Hey everyone! Deepayan this side, a sophomore and open source enthusiast. I am really into web development and currently learning that. Really happy to be a part of this community

**Reply by Hrithik5** [reply]
Hello Everyone! My name is Hrithik. I am currently pursuing my bachelor's degree from India. I am a full-stack developer and an open-source enthusiast. I am currently learning and working on typescript with react. It's great to be a part of this community

**Reply by arushikat13** [reply]
Hi there, I'm Arushi kataria and I am from kota, Rajasthan, India. I'm working as a DevOps engineer trainee at sourcefuse technologies, noida as well I'm a final year undergraduate student in bachelor's of computer science. I'm a folk much passionate about DevOps and python programming . I'm so so happy to get associated with the community and a big thankyou to Eddie for inspiring to be a part of it.

**Reply by eddiejaoude** [reply]
Welcome everyone 🎉

**Comment by eddiejaoude** [comment]
What projects are you working on? 👇

**Reply by eddiejaoude** [reply]
I am working on http://github.com/EddieHubCommunity/api

**Reply by KhushiSharma22** [reply]
I am working on a Personal Note Taking Android App, its called Yola - Collect your thoughts! It focuses much more on personalising the user's experience with their memories :) You can check out the progress here: https://github.com/KhushiSharma22/Yola-Collect-your-thoughts

**Reply by verma-kunal** [reply]
Started working on a Deep Learning project using Tensorflow

**Reply by ItsMeIshan** [reply]
I am currently working on my instagram clone and trying to add some mixed features in it🥳

**Reply by Vyvy-vi** [reply]
Working on learning C and solving problems with C
:)

**Reply by Cahllagerfeld** [reply]
Working on the Eddiehub-API and a Vue3-Composable for Microsoft Teams-Core-Functionality

**Comment by eddiejaoude** [comment]
What are your current challenges? 👇

**Reply by ashwinexe** [reply]
I am stuck in the loop of hunting the just-right organization for me to contribute to. Which leads to a demotivation at times, and unproductivity. 
The general loop cycle goes like this

Pick an org -> Try that application/product -> look at issues and PRs (open and closed) -> realize the tech stack is overwhelming (or) organization is not growing upto my interest -> look for another organization and start over

**Comment by eddiejaoude** [comment]
What are your goals? 👇

**Reply by ItsMeIshan** [reply]
Current goals are
- contribute more and more to awesome projects 
- Connect with awesome folks of the community 🥳

**Reply by Hrithik5** [reply]
Current goals

1. Deep dive into open source world👨‍💻
2. Connet/Network with more and more creative minds.
3. Grow and help others to grow 😉

**Comment by verma-kunal** [comment]


**Reply by eddiejaoude** [reply]
Questions should have their own threads, so people can reply to each one

**Reply by verma-kunal** [reply]
Alright! Thanks Eddie for pointing that out !!

**Reply by eddiejaoude** [reply]
no problem 🤓 , thank you for supporting @verma-kunal

**Comment by GowthamPB** [comment]
How much technical knowledge do you need to take part in Hackathons?
